### PR TITLE
UIOR-459 Cannot save tenant address

### PR DESCRIPTION
**Pull request by AndreiShumski2**
https://issues.folio.org/browse/UIOR-459

### CODE REVIEW

File: src/settings/Addresses.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ class Addresses extends Component {
   onCreate = item => ({
     value: JSON.stringify(item),
     module: moduleName,
-    configName: `${moduleName.toLowerCase()}.${configName}`,
```
Comment (NikitaSedyx) [inline src/settings/Addresses.js]: but `${moduleName.toLowerCase()}.${configName}` === `tenant.addresses`, no?

Comment (AndreiShumski2) [reply src/settings/Addresses.js]: `${moduleName.toLowerCase()}.${configName}` === `tenant.tenant.addresses`

Comment (aliaksei-chumakou) [reply src/settings/Addresses.js]: IMHO, explicit `const configName = 'tenant.addresses';` is better

Comment (NikitaSedyx) [reply src/settings/Addresses.js]: I mean with previous code `${moduleName.toLowerCase()}.${configName}` is `tenant.addresses`

**Pull request merged**